test/util.spec.js

```diff
@@ -514,8 +514,15 @@ describe('UTIL', function() {
             done();
         });
 
+        // it('should return Array of slot names', function(done) {
+        //     // UTIL.generateSlotNamesFromPattern(activeSlot, pattern).should.be.a('array');
+        //     UTIL.isOwnProperty.called.should.be.true;
+        //     UTIL.getIncrementalInteger.called.should.be.true;
+        //     done();
+        // });
+
         it('should return Array of slot names', function(done) {
```

Comment:
reword the test case to use 'string' instead of 'array' since we are checking for string now.